training/8-validating-mcp.md

```diff
@@ -66,6 +66,8 @@ inputSchema: {
 - **`.default(value)`**: Provides a default value if field is missing
 - **Custom error messages**: Pass error message as parameter to constraints
 
+<!-- ? why not just provide the default in only the typescript function? This would cause confusion if someone updates only 1 of the 2... -->
+
 **Note on defaults**: While Zod's `.default()` provides the default value, you should also add a TypeScript default in your function parameter destructuring for type safety:
```

Comment:
why not just provide the default in only the typescript function? This would cause confusion if someone updates only 1 of the 2...